### ISSUE TITLE
The world creation crash is still happening for 1.20.1

### ISSUE THREAD

**Issue by DroidZed** [StellarWind22/Fabric-Shield-Lib#170]
Here's the crash log:
https://pastebin.com/pkUyRksC

**Comment by DroidZed** [comment]
Okay thanks a lot !

**Comment by DroidZed** [comment]
Any news ?

**Comment by DroidZed** [comment]
I have been waiting for a week...can't play my game unless this gets fixed so please respond with anything :(

**Comment by CringeStar** [comment]
Sorry, its exam week for me, so I probably cant look into this until Thursday or Friday, but I'll see what I can do right now.

**Comment by CringeStar** [comment]
Ok I've found the source:

```
    public static final SpecialRecipeSerializer<FabricShieldDecoratorRecipe> FABRIC_SHIELD_DECORATION_SERIALIZER;
    public static final RecipeType<FabricShieldDecoratorRecipe> FABRIC_SHIELD_DECORATION;

    static {
        //Registering Banner Recipe (Lib only)
        FABRIC_SHIELD_DECORATION = Registry.register(Registries.RECIPE_TYPE, new Identifier(MOD_ID, "fabric_shield_decoration"), new RecipeType<FabricShieldDecoratorRecipe>() {
            @Override
            public String toString() {return "test_recipe";}
        });
        FABRIC_SHIELD_DECORATION_SERIALIZER = Registry.register(Registries.RECIPE_SERIALIZER, new Identifier(MOD_ID, "fabric_shield_decoration"), new SpecialRecipeSerializer<>(FabricShieldDecoratorRecipe::new));
        }
```

**Comment by CringeStar** [comment]
I looked up the classes and translated the error:

the lambda `does not define or inherit an implementation of the resolved method 'abstract CraftingRecipe create(Identifier, CraftingRecipeCategory)' of interface SpecialRecipeSerializer$Factory.`

**Comment by CringeStar** [comment]
I'll check with an acutal IDE at home.

**Comment by DroidZed** [comment]
it's okay bro take your time, I just wanted a response nothing else.

**Comment by DroidZed** [comment]
Thanks for the work

**Comment by CringeStar** [comment]
Ok, so I found the issue. In 1.20.1, the specific name of the method is different than in 1.20.2. Since we usually just support the latest update, we went from 1.20, where it worked, to 1.20.2, where it also worked. We never checked 1.20.1 since we assumed we could catch anything different when we went to 1.20.2. So we are going to need to make a 1.20-1.20.1 release and a separate 1.20.2-1.20.4 release, as we did with 1.19.2. It's easy, so hopefully it'll be out by the end of today or tommorow.

**Comment by DroidZed** [comment]
Thank you so much ! I see how things can change between versions and understand it can be confusing 😅
Good work !

**Comment by CringeStar** [comment]
Yep, even little stuff between versions makes it break lol, like going from Registry to Registries forced us to make a new version. I just released the 1.20.1 version on Github, will be on curseforge/modrinth shortly.

**Comment by DroidZed** [comment]
Thanks a lot !!
